fastapi/dependencies/utils.py

```diff
@@ -776,6 +783,19 @@ def get_body_field(*, dependant: Dependant, name: str) -> Optional[ModelField]:
         ]
         if len(set(body_param_media_types)) == 1:
             BodyFieldInfo_kwargs["media_type"] = body_param_media_types[0]
+    
+    if is_form_data(BodyFieldInfo):
+            if importlib.util.find_spec("multipart") is None:
+                logging.error("Import python-multipart.")
+                return
+            else:
+                import multipart as mp
+                if len(mp.__package__) == 0:
+                    logging.error("Wrong multipart import. pip3 uninstall multipart --> pip3 install python-multipart.")
+                    return
+                else:
+                    print('You have the right multipart!')
```

Comment:
If we're okay with this solution, we can clean it up a little bit + throw those Exceptions.